### ISSUE TITLE
Create an Account. With a screen reader, annoying that backspace tells them “blank” on the password field

### ISSUE THREAD

**Issue by yaelberger-commits** [cds-snc/notification-planning#1640]
## Description of issue
With a screen reader, annoying that backspace tells them “blank” on the password field. The user never knows if the password field is empty or not

Found during usability testing with a participant using a screen reader
https://docs.google.com/document/d/1x_AULftkVhmZ8D4uLKeU3Q9KS_HpGqo3HuMSMrkTukY/edit

Describe what is failing
- Screen reader user cannot tell if a field is blank or not
- include an image
- include context

## Potential fix

- What change could fix this issue?
- We could have an indicator of an empty field state
- What nuggets of information could help us fix the issue

## Resources

- Links to documentation or articles that could be helpful
- examples: WCAG docs, ARIA docis, web.dev, Deque, Webaim...

**Comment by yaelberger-commits** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Notify-Planning-614b3ad91bc2030015ed22f5/issues/cds-snc/notification-planning/1640?planning-poker) with Zenhub @amazingphilippe @andrewleith

**Comment by yaelberger-commits** [comment]
Not able to replicate, but we can read documentation around password fields and see if we can improve our code around this

**Comment by smcmurtry** [comment]
@yaelberger-commits tells me they were using JAWS with the Microsoft Edge browser. I tried again with voiceover + Microsoft Edge and was not able to reproduce. I don't have access to JAWS so I can't test that. 

I would recommend closing this issue and to consider working on a "show/hide password" feature at some point in the future.

**Comment by yaelberger-commits** [comment]
Sounds like a good recommendation. I was in the session with this user observing and what happened was whenever they hit backspace on the password field (using JAWS) it said "blank" and it kept doing that whether they still had characters in the field or not, so they never knew if the field was truly empty. It was only this one user we tested it with, but we could use Fable to test with more screenreaders to see if it's a wider issue. 

I agree with the show/hide feature though. That could be a good improvement and I can make a card for that
